sycl/plugins/unified_runtime/ur/adapters/hip/enqueue.cpp

```diff
@@ -1287,6 +1287,7 @@ UR_APIEXPORT ur_result_t UR_APICALL urEnqueueUSMPrefetch(
     ur_queue_handle_t hQueue, const void *pMem, size_t size,
     ur_usm_migration_flags_t flags, uint32_t numEventsInWaitList,
     const ur_event_handle_t *phEventWaitList, ur_event_handle_t *phEvent) {
+#if HIP_VERSION_MAJOR >= 5
```

Comment:
Also adding this check since `hipPointerGetAttribute` is not supported in HIP versions < 5